src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -109,7 +105,12 @@ void CalculatorProgrammerRadixOperators::BitshiftFlyout_Checked(Platform::Object
         announcementString = m_rotateCarryShiftButtonContent;
     }
 
-    this->BitShiftFlyout->Hide();
+    if (announcementString != m_selectedShiftButtonContent)
+    {
+        this->BitShiftFlyout->Hide();
+        m_selectedShiftButtonContent = announcementString;
```

Comment:
I like the idea of using an enum instead of the string content here

Reply:
OK!